pkgs/servers/mautrix-signal/default.nix

```
@@ -2,13 +2,13 @@
 
 python3.pkgs.buildPythonPackage rec {
   pname = "mautrix-signal";
-  version = "unstable-2021-08-12";
+  version = "unstable-2021-11-12";
 
   src = fetchFromGitHub {
     owner = "tulir";
     repo = "mautrix-signal";
```

moved to https://github.com/mautrix/signal

```suggestion
    owner = "mautrix";
    repo = "signal";
```

---

Also should change the homepage for this and `mautrix` and `mautrix-telegram`